tfhe/src/shortint/glwe_compression/parameters.rs

```diff
@@ -0,0 +1,43 @@
+use crate::shortint::parameters::{
+    DecompositionBaseLog, DecompositionLevelCount, DynamicDistribution, GlweDimension,
+    LweDimension, PolynomialSize, StandardDev,
+};
+use std::fmt::Debug;
+
+#[derive(Copy, Clone, Debug, serde::Serialize, serde::Deserialize)]
+pub struct CompressionParameters {
+    pub br_level: DecompositionLevelCount,
+    pub br_base_log: DecompositionBaseLog,
+    pub packing_ks_level: DecompositionLevelCount,
+    pub packing_ks_base_log: DecompositionBaseLog,
+    pub packing_ks_polynomial_size: PolynomialSize,
+    pub packing_ks_glwe_dimension: GlweDimension,
+    pub lwe_per_glwe: usize,
+    pub packing_ks_key_noise_distribution: DynamicDistribution<u64>,
+}
+
+#[derive(Copy, Clone, Debug, serde::Serialize, serde::Deserialize)]
+pub struct PostPbsParmeters {
```

Comment:
something has to indicate this is linked to compression in its name

Reply:
They are not use, will remove

Reply:
Removed